starter-code/contact-app/src/app/contact-list/contact-list.component.html

```diff
@@ -15,4 +15,19 @@
   <input type="text" [(ngModel)]="newContact.image">
 </div>
 
-<button (click)="addContact">Submit New Contact</button>
+<button (click)="addContact()">Submit New Contact</button>
+
```

Comment:
En tus input se habría agradecido que también pusieras el label para dar contexto a tu app: ` <label for="contact-name">Nombre</label>`